scripts/check_minimum_requirements

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
What's the motivation for this script? As long as users (including devs) are `pip install`ing stactools, they shouldn't end up with too-old versions of any of the deps.